libs/pipelines/top_counties_pipeline.py

```diff
@@ -13,10 +13,10 @@
 PROD_BUCKET = "data.covidactnow.org"
 
 TopCountiesPipelineProjectionResult = namedtuple(
-    "TopCountiesPipelineProjectionResult", ["key", "projection_df",]
+    "TopCountiesPipelineProjectionResult", ["projection_df",]
```

Comment:
1. Is the trailing `,` after "projection_df" intentional?
2. I'm not familiar with this code, but creating a tuple with one element feels non-tuple-y.  Can you verify this is intentional / what we want? 😄

Reply:
ya following the pattern of the other pipeline files. I'd rather be consistent here but i aggree it doesn't really make sense?

Reply:
also the formatting is from black

Reply:
Okaydoke!